backend/src/app.ts

```diff
@@ -131,21 +131,15 @@ const isLandlord = (obj: LandlordWithId | ApartmentWithId): boolean => 'contact'
 app.get('/search', async (req, res) => {
   try {
     const query = req.query.q as string;
-    const landlordDocs = (await landlordCollection.get()).docs;
-    const landlords: LandlordWithId[] = landlordDocs.map(
-      (landlord) => ({ id: landlord.id, ...landlord.data() } as LandlordWithId)
-    );
-    const aptDocs = (await buildingsCollection.get()).docs;
-    const apts: ApartmentWithId[] = aptDocs.map(
-      (apt) => ({ id: apt.id, ...apt.data() } as ApartmentWithId)
-    );
+    const landlords = req.app.get('landlords');
+    const apts = req.app.get('apts');
     const aptsLandlords: (LandlordWithId | ApartmentWithId)[] = [...landlords, ...apts];
 
     const options = {
       keys: ['name', 'address'],
     };
     const fuse = new Fuse(aptsLandlords, options);
-    const results = fuse.search(query);
+    const results = fuse.search(query).slice(0, 5);
```

Comment:
This works for now. Later we can change this to include more results and just use pagination.